src/wizard/react-native/index.md

```diff
@@ -17,7 +17,7 @@ npx @sentry/wizard -s -i reactNative
 
 [Sentry Wizard](https://github.com/getsentry/sentry-wizard) will patch your project accordingly, though you can [setup manually](/platforms/react-native/manual-setup/manual-setup/) if you prefer.
 
-- iOS Specifics: When you use Xcode, you can hook directly into the build process to upload debug symbols and source maps. However, if you are using bitcode, you will need to disable the “Upload Debug Symbols to Sentry” build phase and then separately upload debug symbols from iTunes Connect to Sentry.
+- iOS Specifics: When you use Xcode, you can hook directly into the build process to upload debug symbols and source maps. However.
```

Comment:
I think the `However` got left behind.

Reply:
`m`: Does the user need to actively hook this up, or do we patch this? @krystofwoldrich

Reply:
```suggestion
- iOS Specifics: When you use Xcode, you can hook directly into the build process to upload debug symbols and source maps. ```